main-actions/src/main/scala/sbt/ForkTests.scala

```diff
@@ -84,10 +89,13 @@ private[sbt] object ForkTests {
 
             val taskdefs = opts.tests.map(
               t =>
-                new TaskDef(t.name,
-                            forkFingerprint(t.fingerprint),
-                            t.explicitlySpecified,
-                            t.selectors))
+                new TaskDef(
+                  t.name,
+                  forkFingerprint(t.fingerprint),
+                  t.explicitlySpecified,
+                  t.selectors
+              )
+            )
```

Comment:
Close paren `)` dangling almost like close curly bracket looks odd especially for invocation, and this doesn't even line up.

Reply:
And your proposal is to turn that into JSON?

```scala
def main(
    args: Array[
        String
    ]
): Unit = {
  println(
      "Hello, World!"
  )
}
```

Reply:
if the type of `Array` and the argument to `println` are long enough yes: